### PR TITLE
[SYCL] Improve testing of host task

### PR DESCRIPTION
**Pull request by s-kanaev**
A follow-up on #2540.
This patch adds a bit more sophisticated test cases.
The added tests are disabled and should be enabled within patch appropriate issues fix.

### CODE REVIEW

File: sycl/test/host-interop-task/host-task.cpp [hunk 1]
```diff
@@ -15,9 +15,22 @@
...
+// RUNx: %GPU_RUN_PLACEHOLDER %t.out 5
+// RUNx: %ACC_RUN_PLACEHOLDER %t.out 5
+
+// RUNx: %CPU_RUN_PLACEHOLDER %t.out 6
+// RUNx: %GPU_RUN_PLACEHOLDER %t.out 6
+// RUNx: %ACC_RUN_PLACEHOLDER %t.out 6
+
+// RUNx: %CPU_RUN_PLACEHOLDER %t.out 7
+// RUNx: %GPU_RUN_PLACEHOLDER %t.out 7
+// RUNx: %ACC_RUN_PLACEHOLDER %t.out 7
```
Comment (alexbatashev) [inline sycl/test/host-interop-task/host-task.cpp:28]: Why are all these test cases disabled?

**Review by alexbatashev — COMMENTED** [review]
Overall LGTM, but I'd like @DenisBakhvalov and @dm-vodopyanov to take a look as initial reviewers.

**Comment by DenisBakhvalov** [comment]
Please consider splitting the file into separate test-files. Maybe it will be easier to read and understand?

File: sycl/test/host-interop-task/host-task.cpp (hunk 1 above)
Comment (s-kanaev) [reply sycl/test/host-interop-task/host-task.cpp]: The tests are added in disabled state due to existing issue which will lead to hang.
The follow-up patch with issue fix will enable these tests.
I've updated patch description to note that explicitly.

**Comment by s-kanaev** [comment]
> Please consider splitting the file into separate test-files. Maybe it will be easier to read and understand?

Done

**Review by alexbatashev — APPROVED** [review]

**Pull request merged**